app/assets/javascripts/discourse/components/d-editor.js.es6

```diff
@@ -21,6 +21,7 @@ import { wantsNewWindow } from "discourse/lib/intercept-click";
 import { translations } from "pretty-text/emoji/data";
 import { emojiSearch, isSkinTonableEmoji } from "pretty-text/emoji";
 import { emojiUrlFor } from "discourse/lib/text";
+import EmojisStore from "discourse/lib/emojis-store";
```

Comment:
I wonder if this would be better as an ember service that's injected? It seems to be used in components only and that would save a little code.